AustinKellar/PowerOn-Prototype
# Dynamic camera

Goal: Changed the camera so that it follows the character as it moves up the screen

What I changed:
I attached a script to the camera to make it track the player. It's similar to Mario, wherein the camera only moves when the player reaches the upper/lower portion of the screen (currently it's set to a quarter height for both)
I made the player's rigidbody a public variable

Important files:
CameraScript.cs
PlayerMovement.cs


How To Test:
Try making sure that the player is always on-camera
Make sure that when falling, the camera falls at the same rate as the player
Make sure that the code follows the style guide correctly
